### ISSUE TITLE
Show search result section and join plan button only after view similar plan is clicked and not on initial load

### ISSUE THREAD

**Issue by ankit13jain** [CJ8664/wolfpool#50]
Render Create New Plan button next to Join plan button

**Comment by CJ8664** [comment]
Fixed as part of [commit](https://github.com/CJ8664/wolfpool/commit/09651137f0aa13a947b1ec5d1577bbd9e81be909)
